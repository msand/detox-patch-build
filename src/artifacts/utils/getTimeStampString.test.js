const getTimeStampString = require('./getTimeStampString');

describe(getTimeStampString.name, () => {
  it('should format date', () => {
    const date = new Date(Date.UTC(2015, 5, 15, 15, 2, 58));
    expect(getTimeStampString(date)).toMatchSnapshot();
    expect(typeof getTimeStampString()).toBe('string');
  });
});
