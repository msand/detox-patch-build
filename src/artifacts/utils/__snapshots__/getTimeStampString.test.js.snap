// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getTimeStampString should format date 1`] = `"2015-06-15 15-02-58Z"`;
